pages/agent/v3/tokens.md.erb

```diff
@@ -36,7 +36,9 @@ mutation {
 }
 ```
 
-The token description should clearly identify the environment the token is intended to be used for, and is shown on your [Agents page](https://buildkite.com/organizations/-/agents).
+The token description should clearly identify the environment the token is intended to be used for, and is shown on your [Agents page](https://buildkite.com/organizations/-/agents).  
```

Comment:
Maybe an example here?

`Read only token for the static site generator`

I dunno, maybe someone has a better idea?